iModelCore/iModelPlatform/Tests/DgnProject/NonPublished/DgnModel_Test.cpp

```diff
@@ -656,6 +657,41 @@ TEST_F(DgnModelTests, DefinitionModelCreation)
 /*---------------------------------------------------------------------------------**//**
 * @bsimethod
 +---------------+---------------+---------------+---------------+---------------+------*/
+TEST_F(DgnModelTests, SheetIndexModelCreation)

```

Comment:
this test should be done in TypeScript and all of this "handler" stuff should be removed.